src/Todo.js

```diff
@@ -1,7 +1,45 @@
 'use strict';
 
 class Todo {
-  // write code here
+  constructor() {
+    this.listOfItems = [];
+    this.idNumber = -1;
```

Comment:
make it easy as possible
```suggestion
    this.list = [];
    this.uniqueId = -1;
```
```suggestion
    this.items = [];
    this.counter = -1;
```